terraform/modules/hub/files/tasks/static-ingress-fargate.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "name": "static-ingress",
+    "image": "${image_identifier}",
+    "cpu": ${allocated_cpu},
+    "memory": ${allocated_memory},
```

Comment:
non-blocking: these are optional in fargate as the memory is set at the task-level

Reply:
Yeah. It's on my list to tidy these up once the migration to fargate is done.